packages/web-components/fast-foundation/src/tooltip/tooltip.ts

```diff
@@ -395,6 +395,94 @@ export class Tooltip extends FoundationElement {
                 this.horizontalDefaultPosition = this.position;
                 break;
 
+            case TooltipPosition.topLeft:
+                this.verticalPositioningMode = "dynamic";
```

Comment:
Should verticalPositioningMode for all of these be "locktodefault" like the other original positions above? Only the default seems to use dynamic positioning.